setup.py

```diff
@@ -1,13 +1,15 @@
-from setuptools import setup
+from setuptools import setup, find_packages
 import json
 
 with open("metadata.json", encoding="utf-8") as fp:
     metadata = json.load(fp)
 
 setup(
     name='lexibank_gravinachadic',
+    version="0.1",
```

Comment:
Not sure if we should actually add this. Please check with @LinguList or @chrzyki

Reply:
@chrzyki should I erase this line?

Reply:
yes, you can delete the version.